Source/Core/Core/IOS/USB/Emulated/Skylanders/SkylanderFigure.cpp

```diff
@@ -0,0 +1,445 @@
+// Copyright 2023 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "SkylanderFigure.h"
+
+#include <algorithm>
+
+#include <Common/IOFile.h>
+#include <Common/Logging/Log.h>
+#include <Common/Random.h>
+#include <Common/StringUtil.h>
+
+#include <mbedtls/aes.h>
+#include <mbedtls/md5.h>
+
+#include "Core/IOS/USB/Emulated/Skylanders/Skylander.h"
+#include "Core/IOS/USB/Emulated/Skylanders/SkylanderCrypto.h"
+
+namespace IOS::HLE::USB
+{
+u8 PointerToU8(u8* pointer)
+{
+  return *pointer;
+}
+u16 PointerToU16(const u8* pointer)
+{
+  return *(u16*)pointer;
+}
+u16 PointerToU16(u8* pointer)
+{
+  return *(u16*)pointer;
+}
+u32 PointerToU32(u8* pointer)
+{
+  return *(u32*)pointer;
+}
+u64 PointerToU64(u8* pointer)
+{
+  return *(u64*)pointer;
+}
```

Comment:
I don't think having these functions are necessary, when you would call them you could just do
```
*reinterpret_cast<T>(pointer)
```

where `T` is your type you want to convert to.